src/libs/events.js

```diff
@@ -14,6 +14,7 @@ const eventsList = {
   cloudEnvironmentCreate: 'cloudEnvironment:create',
   cloudEnvironmentDelete: 'cloudEnvironment:delete',
   cloudEnvironmentUpdate: 'cloudEnvironment:update',
+  download: 'workspace:data:download',
```

Comment:
maybe rename this to `workspaceDataDownload` and alphabetize?

Reply:
fair enough